### ISSUE TITLE
`jsipfs files cat $HASH` shows no output

### ISSUE THREAD

**Issue by VictorBjelkholm** [ipfs/js-ipfs#476]
To reproduce, run the following in js-ipfs repository

```
$ HASH=$(node src/cli/bin.js files add package.json | cut -f2 -d ' ')
$ node src/cli/bin.js files cat $HASH
```

No output, but should show the contents of package.json

**Comment by VictorBjelkholm** [comment]
Still missing, seems like sub-files from a hash not working correctly either.

Example:

`node src/cli/bin.js files cat /ipfs/QmegvLXxpVKiZ4b57Xs1syfBVRd8CbucVHAp7KpLQdGieC/readme`

Shows no output atm.

**Comment by diasdavid** [comment]
@VictorBjelkholm correct, that is a feature that isn't currently supported, although there is no technical reason why not to, all the pieces are there :)

**Comment by VictorBjelkholm** [comment]
@diasdavid I'll add a new PR for that
